ports/samd/boards/ADAFRUIT_GRAND_CENTRAL_M4_EXPRESS/manifest.py

```diff
@@ -0,0 +1,5 @@
+freeze("$(PORT_DIR)/modules")
```

Comment:
Instead of this line, you better call:
`include("$(PORT_DIR)/boards/manifest.py"`